tg.py

```diff
@@ -2,21 +2,50 @@
 import os
 from itertools import cycle
 
+import openai
+import toml  # type: ignore
 from BingImageCreator import ImageGen  # type: ignore
 from telebot import TeleBot  # type: ignore
 from telebot.types import BotCommand, Message  # type: ignore
 
 from responder import respond_prompt, respond_quota
-from utils import extract_prompt
+from utils import extract_prompt, pro_prompt_by_openai
 
-if __name__ == "__main__":
+
+def main():
     # Init args
     parser = argparse.ArgumentParser()
     parser.add_argument("tg_token", help="tg token")
     parser.add_argument("bing_cookie", help="bing cookie", nargs="+")
+    parser.add_argument(
+        "-c",
+        dest="CONFIG_FILE",
+        help="additional config file",
+        default=None,
+    )
     options = parser.parse_args()
     print("Arg parse done.")
 
+    # Read config
+    config = dict()
+    if options.CONFIG_FILE:
+        with open(options.CONFIG_FILE) as f:
+            config: dict = toml.load(f)
+        print("Config parse done.")
+
+    # Setup openai
+    openai_conf: dict = config.get("openai")
+    if openai_conf:
+        if openai_conf.get("api_key"):
+            openai.api_key = openai_conf.get("api_key")
```

Comment:
for openai_api_key, we need to check the user's env most of us have it in env its more convenient.

Reply:
export OPENAI_API_KEY=xxxxx

Reply:
env var is the default value of openai.api_key

Reply:
See config.sample.toml
It is the most common use case.

Reply:
yep but if user set env, they can not go into our prompt_pro function, they will early reply at here
        if not openai_conf:
            bot.reply_to(message, "OpenAI config not found.")
            prompt_handler(message)

Reply:
From my understanding, If the user runs with `-c`, it will work as intended. Retrieve the key from the env and obtain the `default model` from the configuration file. Otherwise, the `prompt_pro` command won't be activated.
So I think this implementation is ok, use `-c` as the switch for enable openai.

Reply:
make sense let's go